### DISCUSSION TITLE
part-ii/2018-27/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#133] [Question comments]
Part II 2018 Automata and Formal Languages: Paper 1, Section II, G 

https://questions.tripos.org/part-ii/2018-27/

**Comment by artie2000** [comment]
<details>
<summary>(b): I defined by recursion using</summary>

$h(n,k)=\text{succ}(\text{succ}(k))$

</details>

<details>
<summary>(d) (ii): Use</summary>

$\mathbb{K}\oplus\mathbb{N}\setminus\mathbb{K}$.

</details>
